energyplus_rmd/validator.py

```diff
@@ -0,0 +1,57 @@
+from json import loads
```

Comment:
This isn't actually a file, just a rename.  I'm not sure with Git didn't pick up the resemblance.